### ISSUE TITLE
jedi typer script doesn't work

### ISSUE THREAD

**Issue by abrahamneben** [cython/cython#1845]
The [jedi typer script](https://github.com/cython/cython/blob/master/Tools/jedityper.py) doesn't seem to work anymore. Perhaps the jedi api has changed. 

    Traceback (most recent call last):
  File "jedityper.py", line 12, in <module>
    from jedi.parser.tree import Module, ImportName
ImportError: cannot import name 'Module'

**Comment by scoder** [comment]
Yes, it's been unmaintained (and untested) for a while. Patch welcome.
